build/repo.props

```diff
@@ -126,7 +126,6 @@
         <NodeJsProjects Include="
                           $(RepoRoot)src\SignalR\**\*.npmproj;
                           $(RepoRoot)src\Middleware\**\*.npmproj;
-                          $(RepoRoot)src\Components\Browser.JS\**\*.npmproj;
```

Comment:
Why is this being removed from the build?

Reply:
I'm not a 100% certain since I didn't investigate a great deal as to why, but from what I could guess building the SignalR functional test project and the Browser.JS project in parallel would party over the same set of linked directories. However, we don't actually need to build the Browser.JS project on the CI - we have a CodeCheck to verify it is up to date and the generated files are checked in.